apps/chat/src/components/Chat/Publish/PublishWizard.tsx

```diff
@@ -374,13 +374,21 @@ export function PublishModal({
                   {t('Publish to')}
                 </label>
                 <button
-                  className="input-form mx-0 flex grow items-center justify-between rounded border border-primary bg-transparent px-3 py-2 placeholder:text-secondary hover:border-accent-primary focus:border-accent-primary focus:outline-none"
+                  className="input-form mx-0 flex grow items-center rounded border border-primary bg-transparent px-3 py-2 placeholder:text-secondary hover:border-accent-primary focus:border-accent-primary focus:outline-none"
                   onClick={handleFolderChange}
                 >
-                  <span className="truncate">
-                    {constructPath(t(PUBLISHING_FOLDER_NAME), path)}
-                  </span>
-                  <span className="text-accent-primary">{t('Change')}</span>
+                  <div className="flex w-full justify-between truncate whitespace-pre break-all">
+                    <Tooltip
+                      tooltip={constructPath(t(PUBLISHING_FOLDER_NAME), path)}
+                      contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
+                      triggerClassName="truncate whitespace-pre"
+                    >
+                      <span className="truncate">
```

Comment:
why do we have `truncate` twice for trigger?

Reply:
I removed the span